addon/templates/components/polaris-resource-list/filter-control/date-selector.hbs

```diff
@@ -0,0 +1,32 @@
+{{polaris-select
+  label="Select a value"
+  labelHidden=true
+  placeholder="YYYY-MM-DD"
```

Comment:
should this placeholder be something like [here](https://github.com/Shopify/polaris-react/blob/master/src/locales/en.json#L148)

Reply:
Good catch, thanks! 🦅👀